extensions/blocks/slideshow/slideshow.php

```diff
@@ -24,5 +24,161 @@
  */
 function jetpack_slideshow_block_load_assets( $attr, $content ) {
 	Jetpack_Gutenberg::load_assets_as_required( 'slideshow' );
+	if ( class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() ) {
+		return jetpack_slideshow_block_render_amp( $attr );
+	}
 	return $content;
 }
+
+/**
+ * Render slideshow block for AMP
+ *
+ * @param array $attr Array containing the slideshow block attributes.
+ *
+ * @return string
+ */
+function jetpack_slideshow_block_render_amp( $attr ) {
+	static $wp_block_jetpack_slideshow_id = 0;
+	$wp_block_jetpack_slideshow_id++;
+
+	$ids      = empty( $attr['ids'] ) ? array() : $attr['ids'];
+	$autoplay = empty( $attr['autoplay'] ) ? false : $attr['autoplay'];
+	$classes  = array(
+		'wp-block-jetpack-slideshow',
+		'wp-amp-block',
+		'align' . isset( $attr['align'] ) ? $attr['align'] : 'center',
+		$autoplay ? 'wp-block-jetpack-slideshow__autoplay' : null,
+		$autoplay ? 'wp-block-jetpack-slideshow__autoplay-playing' : null,
+	);
+
+	return sprintf(
+		'<div class="%s" id="wp-block-jetpack-slideshow__%s"><div class="wp-block-jetpack-slideshow_container swiper-container">%s%s%s</div></div>',
+		esc_attr( implode( $classes, ' ' ) ),
+		esc_attr( $wp_block_jetpack_slideshow_id ),
```

Comment:
If it's an int, can we switch to `absint`?

```suggestion
		absint( $wp_block_jetpack_slideshow_id ),
```

Reply:
https://github.com/Automattic/jetpack/pull/13009/commits/e4c02deb60601a388ec2f70786a39e451ce4390c